.env.example

```
@@ -300,6 +300,17 @@ NEXT_PUBLIC_AUDIO_MODE_VOICE="alloy"
 # 外部連携モードの有効化（true/false） / Enable external linkage mode (true/false)
 NEXT_PUBLIC_EXTERNAL_LINKAGE_MODE="false"
 
+# 外部連携WebSocket URL / External linkage WebSocket URL
+NEXT_PUBLIC_EXTERNAL_LINKAGE_URL="ws://localhost:8000/ws"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)